docs/testbed/README.testbed.VsSetup.md

```diff
@@ -112,7 +113,26 @@ cd sonic-mgmt
 }
 ```
 
-3. From now on, **all steps are running inside the sonic-mgmt docker**, unless otherwise specified.
+
+
+3. (Optional) Set up proxy: if you get the error `ERROR: unable to find a usable default docker image, please specify one manually` you might need to setup proxy to pull docker image.
+
+   Follows the steps in [Configure docker daemon proxy](https://docs.docker.com/config/daemon/proxy/#daemon-configuration) to setup proxies. You can add these configuration in `etc/docker/daemon.json` and then run `sudo systemctl restart docker` to take effect.
+
+   ```json
+   {
+        ...
+        "proxies": {
+                "http-proxy": "http://proxy.example.com:3128",
+                "https-proxy": "https://proxy.example.com:3129",
+                "no-proxy": "*.test.example.com,.example.org,127.0.0.0/8"
+        }
+   }
+   ```
```

Comment:
Why do we need this configuration? Did you run into the "ERROR: unable to find a usable default ..." error before?

Reply:
If we need a proxy to connect to the internet then `setup-container.sh` would fail since under the hood it's pulling docker-mgmt image: https://github.com/sonic-net/sonic-mgmt/blob/master/setup-container.sh#L139. I ran into this issue when I simulate my environment such that it needs to connect to a proxy to access the internet. 

I decided to add this here even though this is more like a docker issue because our `setup-container` script abstract the docker implementation which might be hard for user to debug.

Reply:
Where did you simulate the env and meet this issue?
I suppose the connectivity to the public internet is straightforward and doesn't need a proxy in our dev-vm?